patches/api/0153-Add-Material-Tags.patch

```diff
@@ -8,6 +8,7 @@ are related to each other by a trait.
 
 Co-authored-by: Jake Potrebic <jake.m.potrebic@gmail.com>
 Co-authored by: Lena Kolb <lenakolb2204@gmail.com>
```

Comment:
you missed the `-` between the `authored` and `by` :) maybe you can add it to the line above too? But another person should confirm this.

Reply:
doh! but should not be too big a deal.

Reply:
I'll wait for a second confirmation first.